src/Components/CreateImageWizard/steps/Users/component/UserInfo.tsx

```diff
@@ -59,6 +72,28 @@ const UserInfo = () => {
           fieldName="userPassword"
         />
       </FormGroup>
+      <FormGroup isRequired label="SSH key">
+        <HookValidatedInput
+          ariaLabel="public SSH key"
+          value={userSshKey || ''}
+          type={'text'}
+          onChange={(_e, value) => handleSshKeyChange(_e, value)}
+          placeholder="Paste your public SSH key"
+          stepValidation={stepValidation}
+          fieldName="userSshKey"
+        />
+        <Button
+          component="a"
+          target="_blank"
+          variant="link"
+          icon={<ExternalLinkAltIcon />}
+          iconPosition="right"
+          href="https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_basic_system_settings/assembly_using-secure-communications-between-two-systems-with-openssh_configuring-basic-system-settings#generating-ssh-key-pairs_assembly_using-secure-communications-between-two-systems-with-openssh"
+          className="pf-u-pl-0"
```

Comment:
This will need a PF version slug to work properly:

```suggestion
          className="pf-v5-u-pl-0"
```